README.md

```diff
@@ -104,9 +104,17 @@ For this option, you need to download the VEP cache, which is used in the `gn-ve
     docker-compose up -d
     ```
 
+#### Compatibility Matrix
+
+| oncokb | oncokb-transcript | gn-spring-boot | gn-mongo | genome-nexus-vep | MySQL  | Redis  | Oncokb Data |
+| ------ | ----------------- | -------------- | -------- | ---------------- | ------ | ------ | ----------- |
+| v3.x.x | v0.9.4            | v1.x.x         | v0.x     | v0.0.x           | v8.x.x | v7.x.x | v4.x.x      |
```

Comment:
I would limit MySQL to `v8.0.X` because in 8.1, mysql native password was deprecated.
And for genome-nexus-vep, I think we can use `v0.0.1` specifically since there's no other versions after that?